packages/components/src/components/SegmentedControl/SegmentedControl.tsx

```diff
@@ -52,7 +44,7 @@ export const SegmentedControlRoot = (
         <ToggleGroupPrimitive.Root
             ref={ref}
             {...rootProps}
-            className={segmentedControlRootStyles}
+            className={styles.root}
```

Comment:
sometimes we use `.root,` others we use `.elementName`, might be good to standardize. I don't have a preference but since `.root` is used most often, I suggest we normalize to that

Reply:
https://app.clickup.com/t/8695ec0ae

Reply:
https://github.com/Frontify/fondue/pull/2012